setup.cfg

```diff
@@ -61,6 +61,11 @@ install_requires =
     websocket_client
     # lxml super SGML parser, needs libxml2-devel and libxslt-devel
     lxml
-setup_requires = 
```

Comment:
unless this is ensured in setup.py and/or pyproject.toml, this may break naive installation

i would recommend using pyproject.toml

Reply:
We're going to make the move to pyproject.toml (and maybe hatch) at once in a separate PR.